.dev/scripts/startProdLikeContainer.sh

```diff
@@ -0,0 +1 @@
+docker compose -f .dev/compose.prodtest.yml -f .dev/compose.backbone.yml --env-file .dev/compose.backbone.env "$@"
```

Comment:
newline at the end of the file. Also missing shebang.

Reply:
Maybe you should remove this file and include the compose command in the readme. This is way to simple to be a bash script.